GitUI/CommandsDialogs/BrowseDialog/FormUpdates.cs

```diff
@@ -88,23 +98,44 @@ private void SearchForUpdates()
 
         }
 
-        void CheckForNewerVersion(string releases)
+        void CheckForNewerVersion(GitHubReleaseInfo release)
         {
-            var versions = ReleaseVersion.Parse(releases);
-            var updates = ReleaseVersion.GetNewerVersions(CurrentVersion, AppSettings.CheckForReleaseCandidates, versions);
-
-            var update = updates.OrderBy(version => version.Version).LastOrDefault();
-            if (update != null)
+            Version newVersion = null;
+            try
             {
-                UpdateFound = true;
-                UpdateUrl = update.DownloadPage;
-                NewVersion = update.Version.ToString();
+                newVersion = new Version(release.TagName);
+            }
+            catch
+            {
+                // ignored
+            }
+            if (newVersion == null)
+            {
+                UpdateFound = false;
+            }
+            else
+            {
+                UpdateFound = CurrentVersion < newVersion;
+            }
+            if (UpdateFound)
+            {
+                _releasePageUrl = release.HtmlUrl;
+                var downloadUrlFormat = "https://github.com/gitextensions/gitextensions/releases/download/{0}/GitExtensions-{0}-Setup.msi";
```

Comment:
We don't want to be bound to a specific hosting, this is the reason why the download url is not hardcoded in the program. We can change the hosting and older version of GitExt still will be notified.